### PR TITLE
feat: improved documentation site and release process

### PR DESCRIPTION
**Pull request by maoo**
Read more on https://finosfoundation.atlassian.net/browse/ODP-52

Preview website on http://fdc3-api.finos.org

### CODE REVIEW

**Comment by rikoe** [comment]
@maoo can you provide a bit more context as to the reason for/changes in this PR?

Do you have an example of how the output will look? It is a bit hard to review without that.

**Comment by maoo** [comment]
> @maoo can you provide a bit more context as to the reason for/changes in this PR?
> 
> Do you have an example of how the output will look? It is a bit hard to review without that.

@rikoe, sure! http://fdc3-api.finos.org

File: README.md [hunk 1]
```diff
@@ -1,13 +1,68 @@
...
-See the [specification draft](/Specification-Draft.MD) for the current working documentation, and the [FDC3 Confluence page](https://finosfoundation.atlassian.net/wiki/spaces/FDC3) for further information.
+Visit the [documentation website](https://fdc3-api.finos.org) for more info.
 
-### Development
+## Release
+The FDC3 API is released under the [FINOS NPMJS Organisation](npmjs.com/org/finos).
 
+On every commit, [semantic release](https://semantic-release.gitbook.io/semantic-release/) will be executed by Travis CI and - based on the commit message - will decide to trigger a release or not.
+
+Release managers can use [commitizen](http://commitizen.github.io/cz-cli/) to simplify their commit process; simply install `npm install -g commitizen` and use `git cz` (instead of `git commit`) to commit your changes.
```
[the remark this answers is not in the document]
Comment (maoo) [reply README.md]: I'll add some more text.

File: package.json [hunk 2]
```diff
@@ -1,17 +1,40 @@
 {
-  "name": "fdc3-api",
-  "version": "1.0.0",
+  "name": "@finos/fdc3-api",
```
[the remark this answers is not in the document]
Comment (maoo) [reply package.json]: Good point. Ok, I'll use the `@types` scope. We can still add `finos` npmjs user as collaborator, which will show up the package in the FINOS Org homepage.

**Comment by rikoe** [comment]
@maoo Looks great. What does the version number on the site signify? (1.1.5).

Arguably the current version of the API spec is 1.0.0-beta or 1.0.0-rc (in semver terms), as it is currently in the review period before 1.0.0 ratification.

**Comment by maoo** [comment]
Thanks @rikoe , glad to hear you like it.

The tagging is the latest tag being released and refers to my personal fork, see https://github.com/maoo/API/releases ; you will also notice that `package.json` does NOT have a `version` field anymore, as it's all managed by semantic release; also the Changelog (what you read in https://fdc3-api.finos.org/CHANGELOG.html) is automatically generated by commit history.

More docs on the release process can be found on https://github.com/maoo/API/blob/master/README.md ; please let me know if there's anything missing there.

The way Travis CI is currently configured is that the website gets updated only when a release occurs; this means that, after merging this PR, there should be few manual configuration steps ([documented on ODP-52](https://finosfoundation.atlassian.net/browse/ODP-52?focusedCommentId=11401&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-11401)) and a pre-release (ie 0.9.9) to publish the documentation.

Hope this clarifies a bit more.

**Pull request merged**